sphinx/writers/latex.py

```diff
@@ -1833,6 +1840,8 @@ def warner(msg):
             hlcode = hlcode.rstrip()[:-14]  # strip \end{Verbatim}
             self.body.append('\n' + hlcode + '\\end{%sVerbatim}\n' %
                              (self.table and 'Original' or ''))
+            if ids:
+                self.body.append('\\let\\SphinxLiteralBlockLabel\empty\n')
```

Comment:
It is true that ` \global\let\SphinxVerbatimTitle\empty` is done from inside `\endVerbatim` (as configured in `sphinx.sty` package). On the other hand the `\let\SphinxLiteralBlockLabel\empty` is inserted by the LaTeX writer after `\end{Verbatim}`.  Thus the two are treated a bit differently. Actually the better way is for both to be reset after the `\end{Verbatim}`. This way no `\global` is needed and possibility of nesting exist (which don't truly exist because after all we can't really nest Verbatim things...). The reason for handling the `\SphinxVerbatimTitle` the way it is is purely because at the time I did the earlier commit which brought it I had not dared looked at `latex.py` but was only working with modifying `sphinx.sty`. I should probably now make an additional change to treat it the same way the `\SphinxLiteralBlockLabel` is handled in the commits of this PR.